docs/projects/lottie/layers/linear_gradient.rst

```diff
@@ -0,0 +1,22 @@
+Linear Gradient Layer
+=====================
+
++---------------------+---------------------------------------------------------------------------+
+|       Property      |                         Path to Property in lottie                        |
++=====================+===========================================================================+
+|       Z depth       |               Depends on ordering of layers in lottie format              |
++---------------------+---------------------------------------------------------------------------+
+|        Amount       |   layers/shape.json -> "ef" -> effects/gFill.json -> effects/opacity.json |
++---------------------+---------------------------------------------------------------------------+
+|     Blend_method    |Not supported                                                              |
++---------------------+---------------------------------------------------------------------------+
+|        Gradient     |    layers/shape.json -> "ef" -> effects/gfill.json                        |
++---------------------+---------------------------------------------------------------------------+
+|       Point 1       |layers/shape.json ->  "ef" -> effects/gFill.json ->  "s"                   |
```

Comment:
There is no param point1 or point2. Only `p1` and `p2` are there. I see that you have added point1 and point2 params, make a NOTE of these 2 things and why have you added these 2.